lib/elixlsx/xml_templates.ex

```diff
@@ -541,7 +541,7 @@ defmodule Elixlsx.XMLTemplates do
     """ <>
       Enum.map_join(stringlist, fn {_, value} ->
         # the only two characters that *must* be replaced for safe XML encoding are & and <:
-        "<si><t>#{minimal_xml_text_escape(value)}</t></si>"
+        "<si><t xml:space=\"preserve\">#{minimal_xml_text_escape value}</t></si>"
```

Comment:
Hate to nitpick a 1-line change, but I think `mix format` now recommends keeping the parentheses (correct me if I'm wrong though?)

Reply:
Think you are right, I didn't actually mean to leave out the parens. Change has been made.